<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-193VCU3" failures="2" tests="11" name="scripts.PrivacyPage_T_C" time="95.616" errors="0" timestamp="2024-05-03T16:48:30 IST" skipped="0">
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyOgUrlRepetition" time="9.574"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyHttpInPageSource" time="7.014">
    <failure message="The following asserts failed:
	expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.PrivacyPage_T_C.verifyHttpInPageSource(PrivacyPage_T_C.java:87)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyHttpInPageSource -->
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyProtocol" time="9.852"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyConsole" time="8.088"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyDescRepetition" time="9.101"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyNetworkStatus" time="8.853"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifySubdomain" time="9.518"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyMetaContent" time="7.194">
    <failure message="The following asserts failed:
	Trust and Transparency - Explore Job Board Privacy Policy | All Jobs Hire Understand Our Privacy Policy for Using Our Website expected [true] but found [false],
	Trust and transparency are core values of our platform. Discover our detailed Privacy Policy outlining our data practices and protection measures. Explore our privacy policy to understand how we safeguard your personal information. We take your privacy seriously. Learn how we protect your data. expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Trust and Transparency - Explore Job Board Privacy Policy | All Jobs Hire Understand Our Privacy Policy for Using Our Website expected [true] but found [false],
Trust and transparency are core values of our platform. Discover our detailed Privacy Policy outlining our data practices and protection measures. Explore our privacy policy to understand how we safeguard your personal information. We take your privacy seriously. Learn how we protect your data. expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at scripts.PrivacyPage_T_C.verifyMetaContent(PrivacyPage_T_C.java:79)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
]]>
    </failure>
  </testcase> <!-- verifyMetaContent -->
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyTitleRepetition" time="7.957"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyPageHeading" time="8.253"/>
  <testcase classname="scripts.PrivacyPage_T_C" name="verifyStaticContent" time="10.212"/>
</testsuite> <!-- scripts.PrivacyPage_T_C -->
